language/regexp/character_classes_spec.rb

```diff
@@ -96,9 +96,11 @@
     "\x7F".match(/[[:ascii:]]/).to_a.should == ["\x7F"]
   end
 
-  it "doesn't match non-ASCII characters with [[:ascii:]]" do
-    ('/[[:ascii:]]/').match("\u{80}").should be_nil
-    ('/[[:ascii:]]/').match("\u{9898}").should be_nil
+  not_compliant_on :opal do
+    it "doesn't match non-ASCII characters with [[:ascii:]]" do
+      ('/[[:ascii:]]/').match("\u{80}").should be_nil
+      ('/[[:ascii:]]/').match("\u{9898}").should be_nil
+    end
   end
```

Comment:
@elia @eregon that's right, unfortunately in this case filtering it out happens too late in the process, the code simply fails to compile.